.github/ISSUE_TEMPLATE/bug-report---error-stack--.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report (`error-stack`)
+about: Create a report to help us improve `error-stack`
+title: "[BUG]"
+labels: A-error-stack, A-libs, C-bug
+assignees: TimDiekmann
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```

Comment:
Probably not relevant for `error-stack`.

What do we want here @TimDiekmann, OS, rustc version, compile target? (I think we should give the commands to get the relevant information in the template, there's probably an example already on the Rust repo or another crate)

Reply:
In addition to the information, @Alfred-Mountfield provided it would also be helpful which feature flags are enabled to reproduce the error